docs/CONFIG-VARS.md

```diff
@@ -42,6 +42,7 @@ Supported configuration variables are listed in the table below.  All variables
 | KUBECONFIG | Path to kubeconfig file | string | | true | | viya |
 | V4_CFG_SITEDEFAULT | Path to sitedefault file | string | | false | When not set [sitedefault](../examples/sitedefault.yaml) is used | viya |
 | V4_CFG_SSSD | Path to sssd file | string | | false | | viya |
+| DEPLOYMENT_OPERATOR_ENABLED | Is the SAS Deployment Operator present in the cluster | bool | false | false | | viya |
```

Comment:
If this value is false then we just use the LifeCycle Operations portion of this tool set? Asking as it's not clear or spelled out here.

_Originally posted by @thpang in https://github.com/sassoftware/viya4-deployment/pull/224#discussion_r939030552_

Reply:
As the viya4-deployment currently stands, it does not automate the installation of the Deployment Operator.
If the user manually installed the deployment operator beforehand they would enable this flag if they wanted to make use of it.

This doc is still a WIP.

Reply:
Still confused on this one. Is the flag set to install the operator? The operator present in the cluster statement is confusing.

Reply:
Like my previous comment stated, this flag does not install the deployment operator. You would enable this flag if you have manually installed the deployment operator beforehand. Enabling this flag currently modifies some of the orchestration generation/deployment steps for Viya deployments that have the DO. This doc is still WIP.

The automated installation of the DO with the this repository is a separate feature/task. This task is just focused on the making use of the orchestration tooling.

Reply:
Once the deployment operator is present in the cluster there is no way to find that out? If that's the case we need to reach out to the DO group and find out how we tell if there is a cluster based DO, a namespace based DO in an existing cluster. Can you verfiy?

Reply:
The deployment operator will be namespace based if it already exists in the namespace we planning on deploying Viya into. It's cluster based (cluster-wide) if it has it's own namespace and and watches for deployments in all namespaces. You can tell if it exists by checking the namespace for a pod/deployment object.

https://go.documentation.sas.com/doc/en/itopscdc/v_030/dplyml0phy0dkr/n137b56hwogd7in1onzys95awxqe.htm

In terms of just this PR, whether it's cluster-wide or in namespace mode the viya deploy command will be the same. It's the install/uninstall of the DO where that differentiation will matter when we add that feature in a following PR.

Reply:
Ok, so if you know what to check, why the flag? You already know the state of the DO in the cluster, correct?

Reply:
@jarpat thoughts on that last question?

Reply:
Missed this question, doc still is still a todo & this is out of date. 
But the flag is now going to `V4_DEPLOYMENT_OPERATOR_ENABLED` it behaves exactly like one of our existing config vars `CERT_MANAGER_ENABLED` it controls whether or not we are going to be deploying DO into the cluster.